dragonfly/block/glazed_terracotta.go

```diff
@@ -0,0 +1,64 @@
+package block
+
+import (
+	"github.com/df-mc/dragonfly/dragonfly/block/colour"
+	"github.com/df-mc/dragonfly/dragonfly/item"
+	"github.com/df-mc/dragonfly/dragonfly/world"
+	"github.com/go-gl/mathgl/mgl64"
+)
+
+// GlazedTerracotta is a vibrant solid block that comes in the 16 regular dye colors.
+type GlazedTerracotta struct {
+	// Colour specifies the colour of the block.
+	Colour colour.Colour
+	// Facing specifies the face of the block.
+	Facing world.Face
```

Comment:
I think `world.Direction` might fit better here. I don't think glazed terracotta can face upwards, can it?